.ci/go-lint.sh

```diff
@@ -25,16 +25,15 @@ then
 fi
 
 gometalinter \
-	--exclude='error return value not checked.*(Close|Log|Print).*\(errcheck\)$' \
-	--exclude='.*_test\.go:.*error return value not checked.*\(errcheck\)$' \
-	--exclude='duplicate of.*_test.go.*\(dupl\)$' \
-	--disable=aligncheck \
-	--disable=gotype \
-	--disable=gas \
-	--disable=vetshadow \
-	--cyclo-over=15 \
 	--tests \
-	--deadline=600s \
 	--vendor \
-	--errors \
+	--disable-all \
```

Comment:
@jodh-intel What's wrong with `--disable-all` ? That means that we have to enable every linter on purpose. I think this is good to know which linter we enabled and what we could expect. Otherwise, they enable some fancy linters and we get a ton of outputs...